src/algokit/cli/generate.py

```diff
@@ -3,12 +3,36 @@
 
 import click
 
+from algokit.core.generate import load_custom_generate_commands
 from algokit.core.typed_client_generation import ClientGenerator
 
 logger = logging.getLogger(__name__)
 
 
-@click.group("generate")
+class GeneratorGroup(click.Group):
+    def get_command(self, ctx: click.Context, cmd_name: str) -> click.Command | None:
+        return_value = super().get_command(ctx, cmd_name)
+
+        if return_value is not None:
+            return return_value
+
+        commands = load_custom_generate_commands(Path.cwd())
+
+        for command in commands:
```

Comment:
what about making `load_custom_generate_commands` return mapping str -> Command, then you can just `commands.get(cmd_name)`?

Reply:
@achidlow good point, refined to return a dict mapping